### ISSUE TITLE
Empty output .pd

### ISSUE THREAD

**Issue by mightypanda** [twardoch/mkdocs-combine#13]
Hello,
I am using mkdocs and mkdocs-combine in a Windows Bash shell, everything works well for mkdocs serve and build, as well as mkdocs-combine pip installation with pandoc.
When i run mkdocs-combine i get an emtpy .pd file without any warning or notices.
Is there a way to have debug info?
Attached is the whole mkdocs project folder, consisting in a very minimal test site.

Thank you!
[mydocs.zip](https://github.com/twardoch/mkdocs-combine/files/2054766/mydocs.zip)

**Comment by altitude1326** [comment]
Same problem on mac, it does not process correctly mkdocs.yml pages hierarchy. It produces a pandoc file with only the uppermost sections.

**Comment by twardoch** [comment]
Since a few months ago, MkDocs introduced the ability to write plugins, I'm going to adopt this code 

https://github.com/shauser/mkdocs-pdf-export-plugin

to replace mkdocs-combine. MkDocs plugins give a much greater reliability that everything is really processed the way it should be.

**Comment by altitude1326** [comment]
Thank you @twardoch !
It work like a charm but create one pdf per markdown.

**Comment by twardoch** [comment]
Yeah, it needs to be adapted but it shows the principle well enough. I'll need to revisit it but if the code works as an MkDocs plugin then it'll have access to the same document tree as MkDocs proper, so combining it shouldn't be a problem. I still plan to just output one large Markdown file but it'll be more reliable, and possibly deal correctly with in-page links, something that the current code could never properly achieve.
